src/convertToObject.js

```diff
@@ -7,6 +7,15 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((arrKeyAndValue) => arrKeyAndValue.split(':'))
+    .filter((item) => item.length > 1)
+    .reduce((objectRes, arrTwoWords) => {
+      objectRes[arrTwoWords[0].trim()] = arrTwoWords[1].trim();
+
+      return objectRes;
```

Comment:
The parameter 'arrTwoWords' is not descriptive and could be misunderstood. Consider renaming it to 'stylePair' to better represent its purpose of storing a key-value pair from a CSS declaration. This aligns with the [CODE STYLE] guideline to use descriptive variable names.